ui/packages/consul-ui/translations/components/consul/en-us.yaml

```diff
@@ -156,9 +156,12 @@ topology-metrics:
       header: No traffic
       body: Add "{upstream}" as an explicit upstream of "{downstream}" or set the "{downstream}" proxy to "transparent" mode to enable traffic.
       action: Documentation
+  status:
+    error: Unable to load metrics
+    loading: Loading Metrics
+    ingress-gateway: Viewing metrics for ingress gateways is not currently supported in the UI.
```

Comment:
We use a mixture of title and capital case here:

```
Unable to load metrics
Loading Metrics
Viewing metrics
```

I'm never sure whether we have a rule to stick to with these, could you check in with design and see whether it should be `Metrics` or `metrics`, if it was me I'd stick to `metrics` unless at the start of a phrase/sentence (I think there might sometimes be an exception for where we capitalize 'Consul Nouns', which makes me wonder where we should use `Ingress Gateways` or `ingress gateways` also.

Ideally we'd have a written rule to follow so we can refer to it if we are unsure.

@trujillo-adam I wondered what your opinion on this was? If you can describe a pattern we should follow I can maybe add it to our Eng Docs somewhere so it's written down.

Reply:
@johncowen I'll update to `metrics` and `Ingress Gateway`. 

If you want to have a full-fledged discussion about capitalization patterns please do so outside out this PR.

Reply:
No problem!

https://github.com/hashicorp/consul/pull/10888

@trujillo-adam if you have anything to add here, we should probably use that ^ PR, thanks!